content/docs/dvclive/index.md

```diff
@@ -0,0 +1,19 @@
+# dvclive
+
+[dvclive](https://cml.dev) is an open-source python library for monitoring
+[dvclive](https://cml.dev) is an open-source python library for monitoring the
+progress of metrics during training of machine learning projects.
+
+dvclive is integrated seamlesly with dvc and logs produced by it can be fed to
+`dvc plots` command. Even though, one does not need dvc to visualize dvclive
+logs, as they are saved into easily parsable tsv format, feel free to apply
+custom visualization methods.
+
+We have created dvclive with two principles in mind:
+
+- **no dependencies** While you can install optional integrations for various
+  frameworks, basic dvclive installation does not need anything besides standard
+  python libs.
+- **integration with DVC** DVC is able to recognize when its being used in
+  tandem with dvclive and is able to provide useful features - like producing
+  training summary during training.
```

Comment:
More copy edits:
```suggestion
We have created Dvclive with two principles in mind:

- **No dependencies.** While you can install optional integrations for various
  frameworks, the basic `dvclive` installation doesn't have requirements besides
  [Python](https://www.python.org/).
- **DVC integration.** `dvc` recognizes when its being used along with
  `dvclive`. This enables useful features automatically, like producing model
  training summaries, among others.
```

Reply:
> producing model
  training summaries, among others

What are other main auto-features of dvc+live?

Reply:
My thoughts: adding checkpoints, logging metrics and plots in DVC-compatible formats.